docs: simplify streaming metadata (#84253)
vercel/next.js: docs/01-app/01-getting-started/14-metadata-and-og-images.mdx

### Streaming metadata

If metadata isn’t immediately ready, Next.js renders and sends the initial UI. As soon as `generateMetadata` resolves its results are streamed to the client. The streamed metadata tags are appended to the `<body>`.

Bots that can’t execute JavaScript are **opted out of this behavior**. Next.js waits for `generateMetadata` before sending the initial UI. The HTML received by these bots contains metadata in the `<head>` tag.

This improves perceived performance by reducing [TTFB](https://developer.mozilla.org/docs/Glossary/Time_to_first_byte) and often [LCP](https://developer.mozilla.org/docs/Glossary/First_contentful_paint) for users and JS-capable bots.

Statically rendered pages don’t use streaming since metadata is resolved at build time.

You can customize which user agents receive blocking metadata, or **disable streaming metadata** completely, with the [`htmlLimitedBots`](/docs/app/api-reference/config/next-config-js/htmlLimitedBots#disabling) option in your Next.js config file.

> Learn more about [Streaming metadata](/docs/app/api-reference/functions/generate-metadata#streaming-metadata).

Author: We can could trim down here, but I think this section is more accessible than the API reference.

In the API reference, the streaming metadata section is under the generateMetadata Reference, which is fairly long

### Streaming metadata

> For dynamically rendered pages, Next.js streams metadata separately, injecting it into the HTML once `generateMetadata` resolves, without blocking UI rendering.

Author: ```suggestion
To prevent blocking the UI for dynamically rendered pages, Next.js streams metadata separately, injecting it into the HTML once `generateMetadata` resolves.
```

Maybe...

> For dynamically rendered pages, if resolving `generateMetadata` delays rendering UI, Next.js streams the resolved metadata separately and injects it into the HTML as soon as it's ready.

Author: ```suggestion
For dynamically rendered pages, Next.js streams metadata separately, injecting it into the HTML once `generateMetadata` resolves, without blocking UI rendering.
```
